src/main/java/uk/gov/pay/api/ledger/service/TransactionSearchService.java

```diff
@@ -111,14 +115,14 @@ private TransactionSearchResults processResponse(Response connectorResponse) {
         );
     }
 
-    private PaymentForSearchResult getPaymentForSearchResult(ChargeFromResponse charge) {
+    private PaymentForSearchResult getPaymentForSearchResult(TransactionResponse charge) {
```

Comment:
maybe rename `charge` to `transaction` or `transactionResponse`?

Reply:
fair enough